simulators/ethereum/eest/consume-engine/Dockerfile

```diff
@@ -1,28 +1,26 @@
 # Builds and runs the EEST (execution-spec-tests) consume engine simulator
-FROM python:3.10-slim
+FROM ghcr.io/astral-sh/uv:python3.10-bookworm-slim
 
 ## Default fixtures and branch
 ARG fixtures=stable@latest
-ENV INPUT=${fixtures}
+ENV FIXTURES=${fixtures}
 ARG branch=main
 ENV BRANCH=${branch}
 
-## Install dependencies
-RUN apt-get update && \
-    apt-get install -y git wget tar && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
 ## Clone and install EEST
-RUN git clone https://github.com/ethereum/execution-spec-tests.git --branch "$BRANCH" --single-branch --depth 1
-WORKDIR execution-spec-tests
-RUN pip install uv && uv sync
+RUN apt-get update && apt-get install -y git && \
+    git clone https://github.com/ethereum/execution-spec-tests.git --branch "$BRANCH" --single-branch --depth 1 && \
```

Comment:
Can you check if there is a non-breaking way to allow any ref here? I think `--branch` only allows branch names and tags but not commit hashes? Could be I'm wrong, plz check. Would be really handy for debugging and testing, see: 
- https://github.com/ethereum/execution-spec-tests/issues/1354